README.md

```diff
@@ -1,60 +1,59 @@
-SlideActionView is a simple widget that provides a nice slide-to-left/right interaction. This is a part of my efforts to modularize some of the things that I write; it was originally a part of [Alarmio](https://jfenn.me/projects/alarmio), and has been separated into its own library.
+# SlideActionView
 
-[![JitPack](https://jitpack.io/v/me.jfenn/SlideActionView.svg)](https://jitpack.io/#me.jfenn/SlideActionView)
-[![Build Status](https://travis-ci.com/fennifith/SlideActionView.svg?branch=master)](https://travis-ci.com/fennifith/SlideActionView)
-[![Discord](https://img.shields.io/discord/514625116706177035.svg?logo=discord&colorB=7289da)](https://discord.gg/ucBGWDy)
+A HMOS library which provides slide-to-left/right interaction.
 
-For testing and experimentation purposes, a sample apk can be downloaded [here](https://jfenn.me/projects/slideactionview).
+## Source
+Inspired by [fennifith/SlideActionView](https://github.com/fennifith/SlideActionView) - version 0.0.2
 
-|Sample App|Use in Alarmio|
-|------|------|
-|![img](./.github/images/sample.png?raw=true)|![img](./.github/images/alert.gif?raw=true)|
+## Feature
+SlideActionView is a simple widget that provides a nice slide-to-left/right interaction.
```

Comment:
In the feature section, mention about the OhosUtils library that you are using internally : https://github.com/applibgroup/OHOSUtils/

Reply:
Done